tests/PaletteTest.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace marijnvdwerf\palette\Tests;
+
+use Intervention\Image\ImageManager;
+use marijnvdwerf\palette\Palette;
+use PHPUnit\Framework\TestCase;
+
+class PaletteTest extends TestCase
+{
+    public function testGenerateEmptySwatches()
```

Comment:
I think this test case would be better if it only looked at the histogram -> palette conversion. That we we'd just need to feed it a small list of colours, and only validate that part of the library. Something like  `$pal = new Palette(Swatch[] $swatches)`, and then validate where the proper ones were picked and generated,

Reply:
Yes, that makes sense. I will try to identify a concise palette where a dark vibrant swatch wouldn't have previously been generated. At the minute I just have a sample set of images missing a swatch and I'm not sure what's consistent with their histograms.